packages/ckeditor5-dom-converter/src/ConversionApi.ts

```diff
@@ -0,0 +1,107 @@
+import { isElement } from "@coremedia/ckeditor5-dom-support/Elements";
+import { isAttr } from "@coremedia/ckeditor5-dom-support/Attrs";
+
+/**
+ * Contextual information and API during DOM conversion.
+ */
+export class ConversionApi {
+  readonly targetDocument: Document;
```

Comment:
Won't adjust now. It has similarities to CKEditor API I reviewed while implementing this. And `sourceNode` may have relevance for custom rules, if the original node disappeared. Thus, one assumption is, for example, that a rule decides to remove the node. Another rule needs information of this removed node, though, and may now look it up in the original DOM instead.

This is not a well-paved path, though, and may need adjustments. E.g., you could argue, that rules should always do checks on the source node rather than on the imported node. Nevertheless, for existing rules, the `sourceNode` is not relevant.